test_runner/fixtures/compare_fixtures.py

```diff
@@ -333,3 +336,29 @@ def neon_with_baseline(request: FixtureRequest) -> PgCompare:
     fixture = request.getfixturevalue(request.param)
     assert isinstance(fixture, PgCompare), f"test error: fixture {fixture} is not PgCompare"
     return fixture
+
+
+@pytest.fixture(scope="function", autouse=True)
+def sync_after_each_test():
+    # some of the benchmarks are quite write-happy, and create issues to start
+    # the processes up in 10s.
+    # SYNC_AFTER_EACH_TEST is only "true" for build_and_test workflow
+    # benchmarks, which run test_runner/performance cases sequentially.
+    # however, this fixture is included for all regress tests as well.
+    #
+    # use the environment variable SYNC_AFTER_EACH_TEST to control whether any
+    # syncing happens.
+    key = "SYNC_AFTER_EACH_TEST"
+    enabled = os.environ.get(key) == "true"
+
+    yield
+
+    if not enabled:
+        # regress test, or running locally
+        return
+
+    start = time.time()
+    # we only run benches on linuxes, the method might not exist on windows
```

Comment:
Well, technically, macOS is not a linux 😄

Reply:
:D

Reply:
```suggestion
    # we only run benches on unices, the method might not exist on windows
```